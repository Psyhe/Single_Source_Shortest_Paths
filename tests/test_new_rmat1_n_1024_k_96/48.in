1024 528 538
528 8 28
528 512 45
528 5 106
528 768 118
528 256 64
528 402 118
528 34 70
528 17 147
528 80 203
528 418 9
528 517 85
528 3 106
528 44 254
528 448 172
528 50 18
528 546 104
528 2 113
528 36 223
528 0 124
528 96 129
528 769 114
528 5 28
528 72 216
528 608 51
528 2 106
528 0 181
528 257 149
528 324 243
528 35 153
528 576 211
528 36 71
528 10 68
528 0 93
528 18 160
528 132 130
528 514 32
528 17 7
528 896 194
528 740 94
528 353 169
528 0 231
528 386 159
528 262 82
528 1 175
528 644 109
528 130 9
528 76 30
528 64 46
528 1 124
528 70 134
528 128 143
528 0 105
528 376 179
528 653 41
528 273 132
528 13 101
528 626 156
528 48 16
528 40 210
528 640 203
528 128 151
528 786 70
528 708 207
528 147 166
528 112 174
528 0 214
528 385 27
528 36 249
528 36 104
528 521 121
528 16 83
528 64 150
528 982 195
528 264 225
528 632 99
528 525 134
528 516 142
528 422 23
528 36 229
528 40 81
528 40 202
528 725 39
528 1 155
528 2 46
528 275 128
528 586 150
528 132 147
529 289 167
529 289 158
529 647 123
529 86 71
529 320 9
529 517 83
529 162 244
529 515 92
529 522 229
529 257 65
529 903 125
529 64 241
529 0 45
529 12 112
529 256 98
529 256 100
529 270 11
529 128 218
529 386 205
529 516 238
529 5 115
529 18 203
529 192 253
529 66 127
529 8 109
529 17 118
529 8 232
529 30 103
529 0 186
529 320 191
529 384 57
529 88 182
530 193 125
530 16 50
530 137 67
530 139 139
530 268 94
530 0 213
530 98 116
530 116 82
530 128 204
530 720 50
530 0 36
530 2 188
530 705 206
530 132 212
530 960 7
530 324 155
530 164 7
530 21 51
530 24 190
530 149 253
530 176 239
530 28 73
530 581 90
530 185 63
530 324 86
530 649 252
530 3 185
530 264 228
530 104 67
530 613 226
530 30 160
530 129 119
530 304 29
530 384 39
530 262 210
530 32 245
530 272 40
531 345 49
531 836 37
531 37 168
531 16 102
531 768 95
531 266 210
531 522 151
531 32 157
531 2 169
531 8 228
531 448 168
531 17 153
532 396 53
532 515 101
532 1 243
532 400 184
532 186 148
532 709 114
532 66 131
532 555 25
532 420 17
532 66 126
532 292 14
532 192 25
532 164 205
532 161 138
532 65 85
532 256 194
532 17 180
532 13 222
532 13 22
532 176 234
532 64 208
532 136 19
532 0 199
532 8 170
532 80 211
532 264 166
532 17 210
532 65 70
533 524 52
533 4 177
533 192 32
533 257 250
533 802 26
533 24 154
533 642 18
533 497 165
533 8 245
533 292 118
533 260 88
533 129 63
533 777 107
534 64 233
534 899 227
534 336 10
534 256 41
534 5 17
534 83 249
534 16 63
534 64 33
534 326 97
534 148 67
534 0 89
534 291 121
534 0 146
534 26 210
534 16 98
534 320 243
534 128 7
534 5 34
535 456 73
536 66 70
536 277 242
536 258 189
536 289 45
536 448 174
536 0 131
536 129 73
536 48 99
536 128 188
536 80 57
536 552 241
536 1 235
536 648 25
536 20 199
536 198 169
536 72 246
536 516 66
536 160 246
536 708 106
536 185 135
536 33 77
536 144 231
536 20 46
536 80 92
536 76 55
536 868 244
537 144 192
537 787 69
537 264 4
537 64 96
537 33 211
537 578 11
537 264 179
537 12 8
537 4 8
537 930 10
537 64 179
537 6 55
537 145 27
537 256 126
538 288 199
538 4 112
538 17 210
538 8 137
538 141 89
538 816 56
538 33 144
538 515 114
538 320 79
538 2 172
