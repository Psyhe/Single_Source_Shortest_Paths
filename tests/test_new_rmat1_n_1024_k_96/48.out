14
16
17
57
19
40
12
83
35
10
83
